Error during: yarn build-release

I am using the following commands to build session-desktop 1.1.2 on my slackware 14.2 gnu/linux:

```
npm install yarn --no-save
yarn install --frozen-lockfile
yarn generate
yarn build-release
```

The first three commands were successful (no errors), but "yarn build-release" created the error:

```
giovanni@slackserver:~/testing/session-desktop/session-desktop-1.1.2$ yarn build-release
yarn run v1.22.4
$ cross-env SIGNAL_ENV=production npm run build -- --config.directories.output=release
npm ERR! missing script: build

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/giovanni/.npm/_logs/2020-07-18T15_18_11_132Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
giovanni@slackserver:~/testing/session-desktop/session-desktop-1.1.2$
```
This is the log file:

```
0 info it worked if it ends with ok
1 verbose cli [ '/home/giovanni/.nvm/versions/node/v10.13.0/bin/node',
1 verbose cli   '/home/giovanni/.nvm/versions/node/v10.13.0/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build',
1 verbose cli   '--',
1 verbose cli   '--config.directories.output=release' ]
2 info using npm@6.4.1
3 info using node@v10.13.0
4 verbose stack Error: missing script: build
4 verbose stack     at run (/home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/lib/run-script.js:155:19)
4 verbose stack     at /home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/lib/run-script.js:63:5
4 verbose stack     at /home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:115:5
4 verbose stack     at /home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:418:5
4 verbose stack     at checkBinReferences_ (/home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:373:45)
4 verbose stack     at final (/home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:416:3)
4 verbose stack     at then (/home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:160:5)
4 verbose stack     at /home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/read-package-json/read-json.js:280:12
4 verbose stack     at /home/giovanni/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/graceful-fs/graceful-fs.js:78:16
4 verbose stack     at FSReqWrap.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:53:3)
5 verbose cwd /home/giovanni/testing/session-desktop/session-desktop-1.1.2
6 verbose Linux 4.4.14
7 verbose argv "/home/giovanni/.nvm/versions/node/v10.13.0/bin/node" "/home/giovanni/.nvm/versions/node/v10.13.0/bin/npm" "run" "build" "--" "--config.directories.output=release"
8 verbose node v10.13.0
9 verbose npm  v6.4.1
10 error missing script: build
11 verbose exit [ 1, true ]

```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "grunt": "grunt",
     "icon-gen": "electron-icon-maker --input=images/icon_1024.png --output=./build",
     "generate": "yarn icon-gen && yarn grunt",
-    "build-release": "cross-env SIGNAL_ENV=production npm run build -- --config.directories.output=release",
+    "build-release": "cross-env SIGNAL_ENV=production electron-builder --config.extraMetadata.environment=production --publish=never --config.directories.output=release",
     "build-module-protobuf": "pbjs --target static-module --wrap commonjs --out ts/protobuf/compiled.js protos/*.proto && pbts --out ts/protobuf/compiled.d.ts ts/protobuf/compiled.js --force-long",
     "clean-module-protobuf": "rm -f ts/protobuf/compiled.d.ts ts/protobuf/compiled.js",
     "build-protobuf": "yarn build-module-protobuf",

